### DISCUSSION TITLE
Dicom dataset directories

### DISCUSSION THREAD

**Discussion by Alisoltan82** [Project-MONAI/tutorials#1628] [Q&A]
Good day,

Currently, I'm working on a dataset of Dicom images and the dataset is arranged as shown in the attached image.

My initial idea was to convert Dicom images to Nifti format, divide the dataset into images, and labels, and considering multiple images for a single label I was planning to stack them horizontally.

I failed to do so in several steps and I believe there is an easier way or tutorial to handle similar datasets, I attached the metadata for a much clearer view.

thanks

[metadata.csv](https://github.com/Project-MONAI/tutorials/files/14122859/metadata.csv)

![Untitled_files](https://github.com/Project-MONAI/tutorials/assets/131820552/1c046d2b-9762-477b-b911-ec9113a832f5)

**Comment by KumoLiu** [comment]
Hi @Alisoltan82,
you can refer to this tutorial, https://github.com/Project-MONAI/tutorials/blob/main/modules/load_medical_images.ipynb.
Thanks.

**Reply by Alisoltan82** [reply]
Thanks @KumoLiu 
I would also add a solution here https://docs.monai.io/en/latest/apps.html “TciaDataset” function that gets the data automatically in image,label format
